core/docker/build.sh

```diff
@@ -11,6 +11,7 @@ Builds the Trino Docker image
 -a       Build the specified comma-separated architectures, defaults to amd64,arm64,ppc64le
 -r       Build the specified Trino release version, downloads all required artifacts
 -j       Build the Trino release with specified Temurin JDK release
+-e       Build the Trino release with the early access JDK release
```

Comment:
Can we extract this from the version (22-ea) to avoid adding yet another flag?

Reply:
We potentially could

Reply:
Done